apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -115,13 +131,26 @@ const SummaryResponses = ({
           </div>
         )}
 
-        {/* Section dropdown */}
-        {hasSections && (
-          <label className={styles.sectionSelector}>
-            {i18n.responsesForClassSection()}
-            <SectionSelector reloadOnChange={true} />
-          </label>
-        )}
+        <div className={styles.headerContainer}>
+          {/* Section dropdown */}
```

Comment:
Remove?

Reply:
Style wise, these comments are kinda all over the file.  I didn't add this one in but rather it was moved from somewhere else.  I feel good leaving this in given the style of the rest of the file.